etc/httpd/global.conf

```diff
@@ -128,8 +128,8 @@ RewriteRule ^/ja/firefox/organizations/?$ http://www.mozilla.jp/business/downloa
 # bug 1138280
 RewriteRule ^/ja/(firefox|thunderbird)/(beta/)?notes/ http://mozilla.jp/$1$2/notes/ [L,R=301]
 RewriteRule ^/ja/mobile/(beta/?)notes/ http://mozilla.jp/android/$1$2/notes/ [L,R=301]
-RewriteRule ^/ja/(firefox|thunderbird)/(\d+\.\d+(\.\d+)*)(beta)?/releasenotes(/?|/.+)$ http://mozilla.jp/firefox/$1/$2$3/releasenotes/ [L,R=301]
-RewriteRule ^/ja/mobile/(\d+\.\d+(\.\d+)*)(beta)?/releasenotes(/?|/.+)$ http://mozillla.jp/firefox/android/$1$2/releasenotes/ [L,R=301]
+RewriteRule ^/ja/(firefox|thunderbird)/((android/)?\d+\.\d+(\.\d+)*(beta)?)/releasenotes(/?|/.+)$ http://mozilla.jp/$1/$2/releasenotes/ [L,R=301]
+RewriteRule ^/ja/mobile/(\d+\.\d+(\.\d+)*(beta)?)/releasenotes(/?|/.+)$ http://mozillla.jp/firefox/android/$1$2/releasenotes/ [L,R=301]
```

Comment:
Actually, I think that we can safely remove line 132 entirely, as `/ja/mobile/` should get redirected to `/ja/firefox/android/` which would then get caught by the single rule on 131. Also, I believe we can simplify the instances of `\d+\.\d+(\.\d+)*` to a more readable `[0-9.]`, and `(/?|/.+)` can be safely simplified to `/?`. How about something like this?
```
RewriteRule ^/ja/(firefox|thunderbird)/((android/)?[0-9.](beta)?)/releasenotes/?$ http://mozilla.jp/$1/$2/releasenotes/ [L,R=301]
```